lib/views/changed-files-count-view.js

```diff
@@ -28,17 +28,13 @@ export default class ChangedFilesCountView extends React.Component {
   }
 
   render() {
-    const label =
-      (this.props.changedFilesCount === 1)
-        ? '1 file'
-        : `${this.props.changedFilesCount} files`;
     return (
       <button
         ref="changedFiles"
         className="github-ChangedFilesCount inline-block"
         onClick={this.handleClick}>
-        <Octicon icon="diff" />
-        {label}
+        <Octicon icon="git-commit" />
+        {`Git (${this.props.changedFilesCount})`}
```

Comment:
> This pull request changes the icon and text in the status bar to say "Git (n)" where n is the number of changed files

Interestingly, when I first saw this my initial thought was that n was the number of git commits... I notice currently we have the number as well as the word "Files" which makes it very clear that the number refers to the changed file count. I wonder if there's a way we could make it more clear with your new version... adding the word "Files" might take up too much space. I think it would even be fine to just drop the file count...

@simurai do you have 💭s?

Reply:
> I think it would even be fine to just drop the file count...

That would be the least confusing, but we'd miss out on informing the user that there are uncommitted changes, which might be important information to keep.

> adding the word "Files" might take up too much space.

![screen shot 2018-09-26 at 2 21 53 pm](https://user-images.githubusercontent.com/378023/46059025-76601c00-c198-11e8-91fc-4dc14e059fff.png)

Hmm.. yeah.

Or how do you like some sort of "notification badge":

![image](https://user-images.githubusercontent.com/378023/46059150-fbe3cc00-c198-11e8-8934-e039dff937ad.png)

Atom IDE uses something similar. It could also be just grey to be a bit more subtle.

Reply:
Sounds good to me! Would we do this UXR prior to merging? Or just prior to releasing?